.changeset/shaggy-singers-promise.md

```diff
@@ -0,0 +1,5 @@
+---
+'@ai-sdk/provider-utils': major
+---
+
+SSE parser bug (CRLF should be handled correctly)
```

Comment:
pls change to `fix(provider-utils): fix SSE parser bug (CRLF)`

Reply:
fixed at [8c7fcc7](https://github.com/vercel/ai/pull/6197/commits/8c7fcc7d14715f0e9b12117fd53b9d002aae898a)